tests/test_resources/test_envs/test_env.py

```diff
@@ -30,6 +30,12 @@ def np_summer(a, b):
     return int(np.sum([a, b]))
 
 
+def torch_import():
+    import torch
```

Comment:
could we use a diff library that's not used in any other test env/function yet to make sure its standalone? aws_cpu fixture now uses a torch docker image